lib/ansible/modules/system/lvg.py

```diff
@@ -32,7 +32,9 @@
     - The module will take care of running pvcreate if needed.
   pesize:
     description:
-    - The size of the physical extent in megabytes. Must be a power of 2.
+    - The size of the physical extent. pesize must be a power of 2, or
+      multiple of 128KiB. pesize can be optionally suffixed by a
+      UNIT (k/K/m/M/g/G), default unit is megabyte.
```

Comment:
please add a note that this feature is available since ansible 2.6

Reply:
Done. Thanks.

Reply:
Sorry to not being clear enough, `version_added` should be used if the arg is new in the specivied version. Since the argument already existed, this is not the case here. Please remove `version_added` again.

But we must inform users that since 2.6, it is possible to use suffixes, when reading the docs. So just append 

~~~yaml
    - The size of the physical extent. pesize must be a power of 2, or
      multiple of 128KiB. Since version 2.6, pesize can be optionally suffixed by a
      UNIT (k/K/m/M/g/G), default unit is megabyte.
~~~

Reply:
Done. Thanks. I should have paid more attention.